contract/wizards/contract_manually_create_invoice.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class ContractManuallyCreateInvoice(models.TransientModel):
+
+    _name = 'contract.manually.create.invoice'
+    _description = 'Contract Manually Create Invoice Wizard'
+
+    invoice_date = fields.Date(string="Invoice Date", required=True)
+    contract_to_invoice_count = fields.Integer(
+        compute="_compute_contract_to_invoice_count"
+    )
+
+    @api.multi
+    def action_show_contract_to_invoice(self):
+        self.ensure_one()
+        if not self.invoice_date:
+            contract_to_invoice_domain = [('id', '=', False)]
+        else:
+            contract_to_invoice_domain = self.env[
+                'contract.contract'
+            ]._get_contracts_to_invoice_domain(self.invoice_date)
+
+        return {
+            "type": "ir.actions.act_window",
+            "name": _("Contracts to invoice"),
+            "res_model": "contract.contract",
+            "domain": contract_to_invoice_domain,
+            "view_mode": "tree,form",
+            "context": self.env.context,
+        }
+
+    @api.depends('invoice_date')
+    def _compute_contract_to_invoice_count(self):
+        self.ensure_one()
+        if not self.invoice_date:
+            self.contract_to_invoice_count = 0
+        else:
+            contract_model = self.env['contract.contract']
+            self.contract_to_invoice_count = contract_model.search_count(
+                contract_model._get_contracts_to_invoice_domain(
```

Comment:
For not having to do the same operation twice (DRY principle), create instead a m2m non stored computed field with this result, and make the count computed field to depend on this one, getting `len` of it.